share/qt/make_windows_icon.sh

```diff
@@ -15,3 +15,11 @@ convert ${ICON_SRC} -resize 32x32 bitcoin-32.png
 convert ${ICON_SRC} -resize 48x48 bitcoin-48.png
 convert bitcoin-16.png bitcoin-32.png bitcoin-48.png ${ICON_DST}
 rm bitcoin-16.png bitcoin-32.png bitcoin-48.png
+#regtest
+ICON_SRC=../../src/qt/res/icons/bitcoin_regtest.png
```

Comment:
is not better to change name to pivx_regtest.png?

Reply:
sure it can be, just holded on the naming convention already used, using pivx_regtest would break it if bitcoin and bitcoin_testnet are used. I would either change them all or keep them as they are. Intention of making this PR was providing launcher for regtest with snap, I was not aware that pivx will/wants to keep regtest just wanted to have icon for regtest installed which differs from current testnet/main net icons/splashscreens.